Simple-PHP-IPAM/lib/backup_admin_destinations.php

```
@@ -426,22 +425,44 @@ function ipam_destinations_handle_post(\PDO $db, string $redirectBase): string
+                audit($db, 'backup.vault_key.sudo_failed', 'vault', null,
+                      "action=$action user=$username");
+                if ($action === 'vault_reveal') {
+                    record_auth_failure($db, 'vault_key_reveal', $clientIp, $username);
+                }
             }
-            audit($db, 'backup.vault_key.sudo_failed', 'vault', null,
-                  "action=$action user=$username");
-            return 'Password does not match — vault-key action refused.';
+
+            page_header('Confirm your identity');
+            $stepUpUserId       = $userId;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$stepUpUserId'. (undefined)

(UnusedLocalVariable)